react/week4/github_user_search/src/githubApi.js

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect, useContext } from "react";
+import { UserSearchContext } from "./userSearch.js";
+
+const UserRepos = () => {
```

Comment:
like my other comment, this file could reflect what it exports and simply be called `UserRepos.js`

Reply:
Ok kasper